cognitedata/reveal
# bug: In Safari the screenshot result is empty

#### Type of change

![Bug](https://img.shields.io/badge/Type-Bug-red) 

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-690

## Description :pencil:


- Disables foreignObjectRendering
- Removed workaround for [this problem](https://github.com/niklasvh/html2canvas/pull/2832) which was also introduced by enabling foreignObjectRendering and is therefore no longer needed.

Note: This will reintroduce https://cognitedata.atlassian.net/browse/REV-589, as it will revert the changes done in https://github.com/cognitedata/reveal/pull/2827.

## How has this been tested? :mag:



Using Reveal Examples. With and without UI, with and without custom resolution. In Chrom & Firefox, and on Android.

## Test instructions :information_source:



See above, or use the [preview docs](https://cognitedata.github.io/reveal-docs-preview/2873/docs/next/examples/cad-basic#creating-screenshots). 

## Checklist :ballot_box_with_check:
